packages/googlesheets/src/Adaptor.js

```diff
@@ -233,6 +233,30 @@ export function getValues(spreadsheetId, range, callback = s => s) {
   };
 }
 
+
+/**
+ * Exposes the googlesheets client for more flexible job writing
+ * @function
+ * @public
+ * @example
+ * fn((state, sheets) => {
```

Comment:
This example is actually correct! I just don't think the implementation honours it correctly.